dt/host/InspectorFrontendHost.js

```diff
@@ -187,7 +229,19 @@ WebInspector.InspectorFrontendHostStub.prototype = {
      */
     append: function(url, content)
     {
-        WebInspector.console.error("Saving files is not enabled in hosted mode. Please inspect using chrome://inspect");
+        const remote = require('remote');
+        const fs = remote.require('fs');
```

Comment:
I was a bit hesitant about that as these files are not wrapped in IIFEs, so e are writing on the global scope here, but you are right - duplicating this code is probably even worse.